spec/rails_helper.rb

```diff
@@ -51,4 +51,5 @@
   # The different available types are documented in the features, such as in
   # https://relishapp.com/rspec/rspec-rails/docs
   config.infer_spec_type_from_file_location!
+  config.include AuthenticationHelper
```

Comment:
I would add this to the top of the file, after the `require` statement on line 25, instead of having it here:

```
include AuthenticationHelper
```